test/index.spec.ts

```diff
@@ -364,14 +364,20 @@ describe('onBuild()', () => {
     expect(existsSync(`.netlify/functions-internal/___netlify-odb-handler/handlerUtils.js`)).toBeTruthy()
   })
 
-  it('writes correct redirects to netlifyConfig', async () => {
+  it('when splitting API routes is disabled, it writes correct redirects to netlifyConfig', async () => {
+    const oldProcessEnv = { ...process.env }
+    process.env.NEXT_SPLIT_API_ROUTES = 'false'
```

Comment:
It would help the snapshots if this was set to `false` and not `true` @pieh. 🤦🏻

Reply:
I missed that too lol 🤦